removed analysis.py is still used in ./mypyc/transform/uninit.py

seen when trying to generate manual pages with 0.790 and python3.9:

I: pybuild base:217: /usr/bin/python3.9 setup.py build 
Traceback (most recent call last):
  File "/packages/tmp/mypy-0.790/setup.py", line 144, in <module>
    from mypyc.build import mypycify
  File "/packages/tmp/mypy-0.790/mypyc/build.py", line 43, in <module>
    from mypyc.codegen import emitmodule
  File "/packages/tmp/mypy-0.790/mypyc/codegen/emitmodule.py", line 41, in <module>
    from mypyc.transform.uninit import insert_uninit_checks
  File "/packages/tmp/mypy-0.790/mypyc/transform/uninit.py", line 5, in <module>
    from mypyc.analysis.dataflow import (
ModuleNotFoundError: No module named 'mypyc.analysis'

apparently the analysis module was removed, however mypyc/transform/uninit.py still uses it

re-adding this module lets the build continue, and then it fails with (building from the release tarball):

mypy/fastparse.py:1260: error: "ExtSlice" has no attribute "dims"
mypy/fastparse.py:1264: error: "Index" has no attribute "value"

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -7,6 +7,7 @@ recursive-include mypy/typeshed *.py *.pyi
 recursive-include mypy/xml *.xsd *.xslt *.css
 recursive-include mypyc/lib-rt *.c *.h *.tmpl *.py *.cc
 recursive-include mypyc/ir *.py
+recursive-include mypyc/analysis *.py
 recursive-include mypyc/codegen *.py
 recursive-include mypyc/irbuild *.py
 recursive-include mypyc/primitives *.py

--- a/setup.py
+++ b/setup.py
@@ -181,7 +181,7 @@ setup(name='mypy',
       packages=[
           'mypy', 'mypy.test', 'mypy.server', 'mypy.plugins', 'mypy.dmypy',
           'mypyc', 'mypyc.test', 'mypyc.codegen', 'mypyc.ir', 'mypyc.irbuild',
-          'mypyc.primitives', 'mypyc.transform'
+          'mypyc.primitives', 'mypyc.transform', 'mypyc.analysis'
       ],
       package_data={'mypy': package_data},
       scripts=['scripts/mypyc'],

